### ISSUE TITLE
Reach study results heat map

### ISSUE THREAD

**Issue by marip8** [ros-industrial/reach#6]
Integrate the code for creating a heat map on the target mesh using reach study results

**Comment by c-salmi** [comment]
Hi, thanks for the great work on this repository! We're using it to quantify some design decision of our new mobile manipulator.

Is there currently a way to create a heatmap similar to [the one from the readme](https://github.com/ros-industrial/reach/blob/master/docs/heat_map_colorized_mesh.png) ? Or is this not integrated ?

**Comment by marip8** [comment]
It's currently not integrated into the repository. I had to write a custom C++ radial basis function interpolation library to interpolate scores at each vertex of the mesh and some other code that I never ended up formalizing. More recently, I implemented the heat map visualization in Python using `scipy` and I was a lot happier with it. I did that work on a client project, so I've asked if I can contribute it back to this repo. I should hear back within the week. I can provide more information on the approach though if you're looking to implement this yourself sooner

**Comment by c-salmi** [comment]
Thanks for the response! I realized that for our usecase a heatmap on the mesh itself might not be the optimal representation after all. We're trying to check the reachability of picking items of a shelf, so I manually created a pcd file with points on the shelf. 

![Screenshot from 2022-11-08 11-03-28](https://user-images.githubusercontent.com/23102757/200782694-5b1476c5-d62f-4dbe-8e64-f86e3b6b11bd.png)

The heatmap visualization on the markers is already quite useful. But I would still be interested in the scipy implementation for future projects, if it's okay to make public! Thanks again.
